src/components/util/multi-and-table/select-groups-table.jsx

```diff
@@ -41,11 +41,14 @@ function SelectGroupsTable({
   });
 
   // Get 10 of the selected groups for table below dropdown, table is paginated so on page change more is fetched.
-  const { data: alreadySelectedGroups } = getSelectedMethod({
-    itemsPerPage: 10,
-    page,
-    id,
-  });
+  const { data: alreadySelectedGroups } = getSelectedMethod(
+    {
+      itemsPerPage: 10,
+      page,
+      id,
+    },
+    { skip: !id }
```

Comment:
It's not clear (to me) what the semantics of `{ skip: !id }` is and what it's supposed to do.

Reply:
This is to avoid executing the query when the id is undefined:
https://redux-toolkit.js.org/rtk-query/usage-with-typescript#skipping-queries-with-typescript-using-skiptoken